Simplenote/SPShadowView.swift

```diff
@@ -0,0 +1,97 @@
+import UIKit
+
+// MARK: - SPShadowView
+//
+// SPShadowView draws outside shadow, while everyting inside is kept transparent
+//
+final class SPShadowView: UIView {
+    private let cornerRadius: CGFloat
+    private let roundedCorners: UIRectCorner
+    private let maskLayer: CAShapeLayer = CAShapeLayer()
+
+    override var bounds: CGRect {
+        didSet {
+            updatePath()
+        }
+    }
+
+    /// Designated Initializer
+    ///
+    ///  - cornerRadius: The radius of each corner oval
```

Comment:
Nipticky: If you add `// - Parameters:` before enumerating the method's parameters, Xcode's **Option + Click** will display them under the title **Parameters**

Reply:
Nice! Didn't know about that, thanks!